lib/modules/decoder/PayloadService.ts

```diff
@@ -3,7 +3,7 @@ import { ask, onAsk } from "kuzzle-plugin-commons";
 import { JSONObject, KDocument } from "kuzzle-sdk";
 import { v4 as uuidv4 } from "uuid";
 
-import { DeviceContent, DeviceSerializer } from "../device";
+import { AdminDeviceContent, DeviceContent, DeviceSerializer } from "../device";
```

Comment:
`AdminDeviceContent` should probably be named something like `DeviceRoutingContent`, or `DeviceProvisionningContent`